External/catch/CMakeLists.txt

```diff
@@ -3,20 +3,25 @@ project(catch_builder CXX)
 # Includes Catch in the project:
 include(ExternalProject)
 
-ExternalProject_Add(
-    catch
-    SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/catch
-    GIT_REPOSITORY https://github.com/philsquared/Catch.git
-    GIT_TAG "v2.1.1"
-    TIMEOUT 10
-    UPDATE_COMMAND ""
-    CONFIGURE_COMMAND ""
-    BUILD_COMMAND ""
-    INSTALL_COMMAND ""
-    LOG_DOWNLOAD ON
-   )
+if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/catch")
```

Comment:
This isn't part of the PR I'm assuming...

Reply:
It is, it prevents the build from making a 'git clone' if the user has manually placed the directory there. Couldn't find a cleaner way to do it with the CMake external dependency command directly.